test/fixtures/menu.ts

```diff
@@ -334,3 +337,12 @@ const cssPopupContent = styled('div', `
 document.addEventListener('DOMContentLoaded', () => {
   document.body.appendChild(setupTest());
 });
+
+const cssOverride = styled('div', `
```

Comment:
Is this unused?

Reply:
No, only for manual test, I'll add a way to toggle it through browser.